lambda_functions/crawl_bobae/lambda_function.py

```
@@ -69,8 +74,25 @@ def lambda_handler(event, context):
                 countGroup = countGroup.text
                 countGroup = countGroup.split("|")
 
-                data["view_count"] = countGroup[0].strip().split(" ")[1]
-                data["like_count"] = countGroup[1].strip().split(" ")[2]
+                view_count = 0
+                try:
+                    view_count = int(
+                        countGroup[0].strip().split(" ")[1].replace(",", "")
+                    )
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)